### DISCUSSION TITLE
Block theme related issues (theme: 2022,2023,2024)

### DISCUSSION THREAD

**Discussion by palashkumer** [getdokan/storegrowth-sales-booster#427] [Bug / Problem]
The following functionality is not working properly: To reproduce the issue, the following steps are given below:

- [ ]  **Issue-01**

### Steps

1. **Activate** the **Free shipping bar** module.
2. Go to **Free shipping bar Setting**—>**Banner Setting**-->**Bar Type**( set **sticky** for better understanding )

### Problems

1. Here, the problem is that if you go to the shop page or product page and add products to the cart, it doesn't update automatically.

See the view given below:
![image](https://github.com/Invizo/storegrowth-sales-booster/assets/106841727/afe9523e-49ff-429b-8cc8-015ffad49658)

- [ ]  **Issue-02**

### Steps

1. **Activate** the **Upsell Order Bump** module.
2. go to the **Create New** button and click on it. Create an order bump.
3. Then,  **Activate** the **Quick Cart** module.
4. Then, go to the shop page
5. Then, add your target product to the cart
6. Then click on the **checkout** button

### Problems

1. Here, the problem is that it appears with a scroll bar.

See the view given below:
![image](https://github.com/Invizo/storegrowth-sales-booster/assets/106841727/623af1d7-abcd-4b16-b739-940af001ca78)

**Comment by imtiaz-pranto** [comment]
Needs Testing. Converting it into discussion for further processing
